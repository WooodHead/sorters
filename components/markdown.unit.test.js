import Markdown from './markdown'
import renderer from 'react-test-renderer'

describe('markdown', () => {
    it('renders', () => {
        const component = renderer.create(<Markdown content="### title"/>)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})