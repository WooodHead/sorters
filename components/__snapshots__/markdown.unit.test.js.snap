// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`markdown renders 1`] = `
<div
  className="markdown"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<h3>title</h3>
    ",
    }
  }
/>
`;
